.github/workflows/build-crates-individually.yml

```diff
@@ -37,11 +37,11 @@ on:
       - '.github/workflows/build-crates-individually.yml'
```

Comment:
This seems like a large change. Half the team goes to a conference at the end of the week, so do we need to make this change now, or can it wait until we have most people back?

Reply:
In the worst case scenario we have at least have fewer people making changes and the snowball is way smaller if something bad happens, and we can revert the commit or apply a fix. Theoretically, we're not changing the behavior on the workflows, but the source of the values (while keeping the actual values).

I'm taking into consideration being able to run all the tests on this kind of changes, on an isolated environment, before reaching main...sadly, we're not there yet (but almost)